app/views/admin/users/_form.html.erb

```diff
@@ -19,11 +19,25 @@
     </div>
     <div class="col-md-6">
       <h3>Neighbourhoods</h3>
-      <p>Grants access to every partner and calendar in these neighbourhoods. Use with care!</p>
-      <%= f.association :neighbourhoods,
-                        label: false,
-                        collection: options_for_neighbourhoods,
-                        input_html: {class: 'form-control select2' } %>
+      <p>You have <%= pluralize @user.neighbourhoods.count, 'neighbourhood' %>.</p>
+
+      <% if f.object.root? %>
```

Comment:
Is there a reason to use this method and not `if current_user.root?`?

Reply:
yeah i think this will allow changing this based on the user you are editing's permissions, not yours

Reply:
are you negating your earlier comment? if so then just remove the earlier comment.

Reply:
no, the comments are in agreement, the first one is the method this should be using and the second one is why it's wrong